### PR TITLE
[PLATFORM-1076][PLATFORM-1070] .com → .network

### PR DESCRIPTION
**Pull request by mondoreale**
I've gone through the code and replaced `www.streamr.com` with `streamr.network`. Left out email addresses and other streamr.com subdomains.

Additionally
- I updated links in the footer so we no longer show a link to FAQ and other pages from the old pub site
- Fixed favicons and the image used for sharing
- Updated `userpages/Avatar` story which includes the fixed knob and a different picture URL

### CODE REVIEW

File: app/src/links.js [hunk 1]
```diff
@@ -14,11 +14,6 @@ module.exports = {
     site: {
         network: formatExternalUrl(streamrRoot, 'learn/network'),
     },
-    faq: 'https://www.streamr.com/faq',
-    aboutUs: 'https://www.streamr.com/about',
-    howItWorks: 'https://www.streamr.com#howItWorks',
-    streamrSystem: 'https://www.streamr.com#streamrSystem',
-    tryTheEditor: 'https://www.streamr.com#tryTheEditor',
     contributionSummary: 'https://s3.amazonaws.com/streamr-public/Crowdcontribution+Information+Summary+(20170912).pdf',
```
Comment (juhah) [inline app/src/links.js]: Email addresses in this file still with `streamr.com` domain:
```
     contact: {
        general: 'mailto:contact@streamr.com',
        media: 'mailto:media@streamr.com',
        jobs: 'mailto:jobs@streamr.com',
        labs: 'mailto:labs@streamr.com',
    },
```
Also in `definitions.json`:
```
    "contactGeneral": "mailto:contact@streamr.com",
    "contactJobs": "mailto:jobs@streamr.com",
    "contactLabs": "mailto:labs@streamr.com",
    "contactMedia": "mailto:media@streamr.com",
```

File: app/src/docs/content/streams/workWithStreamsViaApi.mdx [hunk 2]
```diff
@@ -9,12 +9,12 @@ While the Core UI can do just about everything you'd want to do with streams, so
...
 
 <CodeSnippet language='bash'>
-    {`curl -i -X POST -H "Authorization: Bearer MY-SESSION-TOKEN" -d "{\"foo\":\"hello\",\"bar\":24.5}" https://www.streamr.com/api/v1/streams/MY-STREAM-ID/data`}
+    {`curl -i -X POST -H "Authorization: Bearer MY-SESSION-TOKEN" -d "{\"foo\":\"hello\",\"bar\":24.5}" https://streamr.network/api/v1/streams/MY-STREAM-ID/data`}
 </CodeSnippet>
 
 The following endpoint would return the 5 most recent messages in a stream (or to be more precise, the default partition 0 of a stream):
 
-<CodeSnippet language='json'>{`https://www.streamr.com/api/v1/streams/{id}/data/partitions/0/last?count=5`}</CodeSnippet>
+<CodeSnippet language='json'>{`https://streamr.network/api/v1/streams/{id}/data/partitions/0/last?count=5`}</CodeSnippet>
```
Comment (juhah) [inline app/src/docs/content/streams/workWithStreamsViaApi.mdx]: There is one reference in this file to `https://api-explorer.streamr.com`. Also in `apiExplorer.mdx` :)

File: app/src/links.js (hunk 1 above)
Comment (mondoreale) [reply app/src/links.js]: As stated in [the ticket](https://streamr.atlassian.net/browse/PLATFORM-1076):
> _**Do not** update email addresses, as streamr.network emails don't work yet._

File: app/src/docs/content/streams/workWithStreamsViaApi.mdx (hunk 2 above)
Comment (mondoreale) [reply app/src/docs/content/streams/workWithStreamsViaApi.mdx]: First we have to migrate https://api-explorer.streamr.com to https://api-explorer.streamr.network. That's why left it out.

**Pull request merged**